lib/policy.js

```diff
@@ -281,6 +282,41 @@ module.exports = function() {
 
     }, // validateContainerSize
 
+    validateHealthCheck: function(policy, input, msgs) {
+
+      var isDisallowed = (policy.healthcheck)
+        ? !!policy.healthcheck.disallowed
+        : false;
+
+      var healthcheck = (input.ContainerConfig && input.ContainerConfig.Healthcheck)
+        ? input.ContainerConfig.Healthcheck
+        : null;
+
+      // Dockerfile spec outlines a "none" option
+      var healthTest = (healthcheck && healthcheck.Test && Array.isArray(healthcheck.Test))
+        ? healthcheck.Test
+        : null;
+
+      var noneSpecified = (healthTest && healthTest[0].toLowerCase() === 'none');
+      var noCheck = (!healthcheck || noneSpecified);
+
+      if (noCheck) {
+        msgs.addSuccess('no healthcheck specified');
+        return true;
+      }
+
+      if (isDisallowed) {
+
```

Comment:
no whitespace / remove closing comment? or add whitespace and closing comment to 303-306?